students/Matt_Hudgins/Lesson03/list_lab.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+
+
+'''
+    File Name: list_lab.py
+    Author: Matt Hudgins
+    Date created: 5/5/18
+    Date last modified: 5/5/18
+    Python Version 3.6.4
+'''
+
+# Series 1
+
+print("Starting Series 1!")
+
+# Fruit List
+fruit = ["Apples", "Pears", "Oranges", "Peaches"]
+print(fruit)
+
+# User will add a new fruit
+new_fruit = input("What fruit would you like to add?")
+fruit.append(new_fruit)
+print("This is the new list", fruit)
+
+# User will pick a number
+pick_a_number = int(input("Pick a number between 1-4:"))
+print(pick_a_number, fruit[pick_a_number - 1])
+
+# User will add a fruit to the beginning of the list
+add_a_fruit = input("What fruit would you like to add to the beginning of the list?")
+fruit = [add_a_fruit] + fruit
+print(fruit)
+
+# User will insert fruit into list at the beginning
+insert_a_fruit = input("What fruit would you like to add to the beginning?")
+fruit.insert(0, insert_a_fruit)
+print("Updated List:", fruit)
+
+# Searching for fruits that start with the letter P
+for n in fruit:
+    if n.startswith("P"):
+        print("Here's all of the fruit that start with the letter P:", n)
+
+
+# Series 2
+
+
+print("Starting Series 2!")
+
+# This prints and updated list for the user
+print("This is the most upto date list:", fruit)
+
+# This removes the last furit from the list
+del fruit[6]
```

Comment:
how could you make this more dynamic to be able to delete the fruit without knowing its size?